namespace/transitive-dependencies.adoc

```diff
@@ -1,84 +1,127 @@
+jakarta.resource-api-{version}jar
+jakarta.security.auth.message-api-{version}jar
+jakarta.security.enterprise-api-{version}jar
+jakarta.security.jacc-api-{version}jar
+jakarta.servlet.jsp.jstl-api-{version}jar
+jakarta.servlet.jsp-api-{version}jar
+jakarta.servlet-api-{version}jar
+jakarta.transaction-api-{version}jar
+jakarta.validation-api-{version}jar
+jakarta.websocket-api-{version}jar
+jakarta.websocket-client-api-{version}jar
+jakarta.ws.rs-api-{version}jar
+jakarta.xml.bind-api-{version}jar
+jakarta.xml.registry-api-{version}jar
+jakarta.xml.rpc-api-{version}jar
+jakarta.xml.soap-api-{version}jar
+jakarta.xml.ws-api-{version}jar
+javax.inject-{version}jar
+```
+
+1. activation(`javax.activation`)
+2. annotation(`javax.annotation`)
+3. batch(`javax.batch`)
+    * javax.inject(`javax.inject`)
+    * cdi(`javax.decorator`,`javax.enterprise`)
+4. ejb(`javax.ejb`,`javax.xml.rpc.handler`)
+    * xml.rpc(`javax.xml.rpc`)
+    * transaction(`javax.transaction`)
+5. el(`javax.el`)
+6. enterprise.cdi(`javax.decorator`,`javax.enterprise`)
+    * el(`javax.el`)
+    * interceptor(`javax.interceptor`)
+    * javax.inject(`javax.inject`)
+7. enterprise.concurrent(`javax.enterprise.concurrent`)
+8. enterprise.deploy(`javax.enterprise.deploy`)
+9. faces(`javax.faces`)
+    * servlet(`javax.servlet`)
+    * servlet.jsp(`javax.servlet.jsp`)
+    * servlet.jsp.jstl(`javax.servlet.jsp.jstl`)
+    * websocket(`javax.websocket`)
+    * el(`javax.el`)
+    * cdi(`javax.decorator`,`javax.enterprise`)
+    * validation(`javax.validation`)
+    * json(`javax.json`)
+    * ejb(`javax.ejb`,`javax.xml.rpc.handler`)
+10. interceptor(`javax.interceptor`)
+    * annotation(`javax.annotation`)
+    * ejb(`javax.ejb`,`javax.xml.rpc.handler`)
+11. jms(`javax.jms`)
+12. json(`javax.json`)
+13. json.bind(`javax.json.bind`)
+    * json(`javax.json`)
+14. jws(`javax.jws`)
+15. mail(`javax.mail`)
+    * activation(`javax.activation`)
+16. management.j2ee(`javax.management.j2ee`)
+    * ejb(`javax.ejb`,`javax.xml.rpc.handler`)
+17. persistence(`javax.persistence`)
+18. resource(`javax.resource`)
+    * transaction(`javax.transaction`)
+19. security.auth.message(`javax.security.auth.message`)
+20. security.enterprise(`javax.security.enterprise`)
+    * servlet(`javax.servlet`)
+    * interceptor(`javax.interceptor`)
+    * el(`javax.el`)
+    * ejb(`javax.ejb`,`javax.xml.rpc.handler`)
+    * enterprise.cdi(`javax.decorator`,`javax.enterprise`)
+    * security.auth.message(`javax.security.auth.message`)
+    * security.jacc(`javax.security.jacc`)
+21. security.jacc(`javax.security.jacc`)
+    * servlet(`javax.servlet`)
+22. servlet(`javax.servlet`)
+23. servlet.jsp(`javax.servlet.jsp`)
+    * servlet(`javax.servlet`)
+    * el(`javax.el`)
+24. servlet.jsp.jstl(`javax.servlet.jsp.jstl`)
+    * servlet(`javax.servlet`)
+    * servlet.jsp(`javax.servlet.jsp`)
+    * el(`javax.el`)
+25. transaction(`javax.transaction`)
+    * cdi(`javax.decorator`,`javax.enterprise`)
```

Comment:
We also have a dependency on https://github.com/eclipse-ee4j/interceptor-api from https://github.com/eclipse-ee4j/jta-api/blob/master/src/main/java/javax/transaction/Transactional.java#L20
```
$ git remote get-url upstream
git@github.com:eclipse-ee4j/jta-api.git

$ find . -name \*.java -exec grep "import javax" {} \;
import javax.transaction.xa.XAResource;
import javax.enterprise.util.Nonbinding;
import javax.interceptor.InterceptorBinding;
import javax.enterprise.context.NormalScope;
```

Reply:
Yes, it's a transitive dependency of `cdi`.
The only dependency declared in `jta-api` is `cdi`.
I think it's necessary to check all the specs, there are some specs mixed dep's groupId `jakarta.*` and `javax.*`
I will double check all the specs manually and create PRs later.

Reply:
There would be an argument to make it a direct dependency of transaction API in it's pom given the class is used directly in that project.

In other words, if CDI stopped needing interceptor-api, jta-api may still require it to compile.

If you think it is worth an issue feel free to raise it over here: https://github.com/eclipse-ee4j/jta-api